cluster-autoscaler/main.go

```diff
@@ -212,6 +212,7 @@ var (
 	minReplicaCount                    = flag.Int("min-replica-count", 0, "Minimum number or replicas that a replica set or replication controller should have to allow their pods deletion in scale down")
 	nodeDeleteDelayAfterTaint          = flag.Duration("node-delete-delay-after-taint", 5*time.Second, "How long to wait before deleting a node after tainting it")
 	scaleDownSimulationTimeout         = flag.Duration("scale-down-simulation-timeout", 5*time.Minute, "How long should we run scale down simulation.")
+	parallelScaleDown                  = flag.Bool("parallel-scale-down", false, "Whether to allow parallel scale down of nodes.")
```

Comment:
I'd call this `--parallel-drain` to avoid confusion, since parallel scale down is already possible (for empty nodes).